app/components/directives/settings/networkcreateform.ts

```diff
@@ -5,6 +5,7 @@
 import {Component, Input, Output, EventEmitter, OnInit, NgZone} from "@angular/core";
```

Comment:
I think the logout is not clearing the forms that are stored in wizardservice.ts. Will fix it.